sites/partners/src/listings/PaperListingForm/sections/ApplicationAddress.tsx

```diff
@@ -491,33 +491,56 @@ const ApplicationAddress = ({ listing }: ApplicationAddressProps) => {
-                  watch={watch}
-                  defaultDate={{
-                    month:
-                      moment(new Date(listing?.postmarkedApplicationsReceivedByDate))
-                        .utc()
-                        .format("MM") ?? null,
-                    day:
-                      moment(new Date(listing?.postmarkedApplicationsReceivedByDate))
-                        .utc()
-                        .format("DD") ?? null,
-                    year:
-                      moment(new Date(listing?.postmarkedApplicationsReceivedByDate))
-                        .utc()
-                        .format("YYYY") ?? null,
-                  }}
-                />
-              </ViewItem>
-            </GridCell>
-          )}
+          <GridCell className={"mt-4"}>
+            {postmarksConsidered === YesNoAnswer.Yes && (
+              <DateField
+                label={t("listings.postmarkByDate")}
+                name={"postmarkByDateDateField"}
+                id={"postmarkByDateDateField"}
+                register={register}
+                watch={watch}
+                defaultDate={{
+                  month: listing?.postmarkedApplicationsReceivedByDate
+                    ? moment(new Date(listing?.postmarkedApplicationsReceivedByDate)).format("MM")
+                    : null,
+                  day: listing?.postmarkedApplicationsReceivedByDate
+                    ? moment(new Date(listing?.postmarkedApplicationsReceivedByDate)).format("DD")
+                    : null,
+                  year: listing?.postmarkedApplicationsReceivedByDate
+                    ? moment(new Date(listing?.postmarkedApplicationsReceivedByDate)).format("YYYY")
+                    : null,
+                }}
+                dataTestId={"postmark-date-field"}
+              />
+            )}
+          </GridCell>
+          <GridCell className={"mt-4"}>
+            {postmarksConsidered === YesNoAnswer.Yes && (
+              <TimeField
+                label={t("listings.postmarkByTime")}
+                name={"postmarkByDateTimeField"}
+                id={"postmarkByDateTimeField"}
+                register={register}
+                watch={watch}
+                defaultValues={{
+                  hours: listing?.postmarkedApplicationsReceivedByDate
+                    ? moment(new Date(listing?.postmarkedApplicationsReceivedByDate)).format("hh")
+                    : null,
+                  minutes: listing?.postmarkedApplicationsReceivedByDate
+                    ? moment(new Date(listing?.postmarkedApplicationsReceivedByDate)).format("mm")
+                    : null,
+                  seconds: listing?.postmarkedApplicationsReceivedByDate
+                    ? moment(new Date(listing?.postmarkedApplicationsReceivedByDate)).format("ss")
+                    : null,
+                  period:
+                    new Date(listing?.postmarkedApplicationsReceivedByDate).getHours() >= 12
```

Comment:
Rather than create a new moment object each time we're formatting, can you create it once and then call format on it in these places.

Reply:
nice catch, updated